src/include/platform/internal/GenericPlatformManagerImpl.cpp

```diff
@@ -121,7 +121,7 @@ CHIP_ERROR GenericPlatformManagerImpl<ImplClass>::_InitChipStack()
 }
 
 template <class ImplClass>
-CHIP_ERROR GenericPlatformManagerImpl<ImplClass>::_Shutdown()
+CHIP_ERROR GenericPlatformManagerImpl<ImplClass>::_TeardownChipStack()
```

Comment:
Is this name change actually useful?  It seems like this is increasing the size of the delta without introducing a functional difference.  Seems like a recipe for unnecessary pain and merge conflicts.

Reply:
+1

Reply:
Seems like there is a more substantive discussion on this here: https://github.com/project-chip/connectedhomeip/pull/7478/files#r648019733.